ping.go

```diff
@@ -137,6 +137,13 @@ type Pinger struct {
 	// Number of duplicate packets received
 	PacketsRecvDuplicates int
 
+	// Round trip time statistics
+	MinRtt    time.Duration
+	MaxRtt    time.Duration
+	AvgRtt    time.Duration
+	StdDevRtt time.Duration
```

Comment:
Should these remain unexported? I don't think it's intended that these be set directly, only accessed via `Statistics()`. Otherwise there could be a data race if I'm understanding this correctly.

Reply:
You're right, I'll update the PR.